src/test/scala/is/hail/methods/gqDpStatsSuite.scala

```diff
@@ -44,8 +44,7 @@ class gqDpStatsSuite extends SparkSuite {
 
     dpSampleR.foreach {
       case (s, a) =>
-        //        println("Mean: Computed=%.2f, True=%.2f | Dev: Computed=%.2f, True=%.2f".format(a(1).asInstanceOf[Double], sampleMeans(s), a(2)
-        //          .asInstanceOf[Double], sampleDevs(s)))
+//        println("Mean: Computed=%.2f, True=%.2f | Dev: Computed=%.2f, True=%.2f".format(a.dpSC.mean, sampleMeans(samplePositionMap(s)), a.dpSC.stdev, sampleDevs(samplePositionMap(s))))
```

Comment:
delete

Reply:
I want to leave these in, this test in particular is almost impossible to debug.